apb.yml

```diff
@@ -100,7 +100,7 @@ _apb_businesscentral_hostname: &_apb_businesscentral_hostname
 _apb_controller_global_discovery: &_apb_controller_global_discovery
```

Comment:
@mramendi Only comment is that it's "KIE Server" or "KIE Servers" capitalized. If it's already "KIE server" everywhere else, then I guess whatever. Just fyi.

Also, as a heads-up, we will NOT be unifying Process Server/Decision Server to KIE Server this release as we had hoped. It's fine as it is here though and in file names, just leave everything however it is normally. The prod team was not able to address prod impact for 7.5,  so we bumped to 7.6. Last you and I spoke I said I'd let you know if we would do so for any impact in the templates, but said to press on in the absence of news. So I'm confirming that there is now change on server terminology this release.

Reply:
The lines I changed already had "KIE server" and I really don't want to push on alignment, given this is getting deprecated anyway.